smoketest/tests.py

```diff
@@ -634,13 +634,18 @@ class HeaderTestResult(TestResult):
 
     @property
     def description(self):
-        return u'{0} header was {1}'.format(
-            self.test.header,
-            self._get_header_value_from_response(),
-        )
+        header = self.test.header
+        header_value = self._get_header_value_from_response()
+        if header_value is None:
+            description = u'{0} header was not present'.format(header)
+        else:
+            description = u'{0} header was {1}'.format(header, header_value)
+        return description
 
     def __nonzero__(self):
         actual_value = self._get_header_value_from_response()
```

Comment:
Reading this section, specifically looking up what `self._get_header_value_from_response()` does, made me wonder whether we should change `self.response.headers.get(self.test.header)` to `self.response.headers.get(self.test.header, None)` or `self.response.headers.get(self.test.header, False)`.

And, to that end, whether we should audit the rest of the .get()s in this file. It seems like we have a mixture of behavior currently.